.github/optimize/scripts/update-zeebe-identity.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
As mentioned in [this message](https://camunda.slack.com/archives/C09PLKZG42H/p1762895119225539) Monorepo release process has [this workflow](https://github.com/camunda/camunda/blob/main/.github/workflows/update-identity.yml) used to set identity version, which we can try to use it here.